common/djangoapps/util/enterprise_helpers.py

```diff
@@ -29,6 +33,23 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS
 
 
+def consent_needed_for_course(user, course_id):
+    if not enterprise_enabled():
+        return False
+    return consent_necessary_for_course(user, course_id)
```

Comment:
So, right now, enterprise is a regular Django app that gets installed directly. But, eventually, it'll be an IDA we access with a REST API. Having these kinds of aliases right now allows us to have a centralized set of the resources we'll need to update for remote access.